### PR TITLE
i18n: Calendar

### LINKED ISSUE #78: i18n: Calendar

### PR DESCRIPTION
**Pull request by amanuela97**
Resolves #78  

Demo Link: https://wedance-amanuela97.netlify.app/events/

### CODE REVIEW

**Review by best-lucky1030 — COMMENTED** [review]
Good.  👍

**Comment by razbakov** [comment]
@amanuela97 this branch now contains too many changes (including library translations). Please have a look.

File: pages/feed.vue [hunk 1]
```diff
@@ -16,7 +16,7 @@
         ></textarea>
         <div class="flex justify-between">
           <TInputSelectSmall v-model="postType" :options="postTypeList" />
-          <TButton @click="send" title="post a message">Send</TButton>
+          <TButton title="post a message" @click="send">Send</TButton>
```
Comment (best-lucky1030) [inline pages/feed.vue:19]: just changing the position?

**Comment by razbakov** [comment]
@amanuela97 you merged demo branch into this one, could you please remove unrelated commits from this branch?

**Pull request merged**